README.md

```diff
@@ -44,14 +44,15 @@ pip install mnelab
 The `mnelab` command starts the application.
 
 #### Via conda
-An (unofficial, but regularly updated) conda package can be installed from [conda-forge](https://conda-forge.org/), ideally into
-its own dedicated environment:
+An (unofficial, but regularly updated) conda package can be installed from [conda-forge](https://conda-forge.org/).
+We **strongly** suggest to install `mnelab` into its own, dedicated environment to ensure smooth installation
```

Comment:
I'd remove the comma between "own, dedicated"

Reply:
Done in 93d6870